modules/tensor_mechanics/doc/content/modules/tensor_mechanics/FractureIntegrals.md

```diff
@@ -106,3 +106,22 @@ The $T$-stress is the first second-order parameter in Williams' expansion of str
 The MOOSE implementation of the capability to compute these fracture integrals is provided using a variety of MOOSE objects, which are quite complex to define manually, especially for 3D simulations. For this reason, the to compute $J$-integrals or interaction integrals, the [DomainIntegral Action](/DomainIntegralAction.md) should be used to set up all of the required objects.
 
 !listing modules/tensor_mechanics/test/tests/j_integral/j_integral_3d.i block=DomainIntegral
+
+
+## C-Integral
+
+The C(t) integral can be described as a path integral that is obtained by replacing displacements and strains with displacement rates and strain rates
```

Comment:
The C(t) integral is a fracture integral that is obtained in essentially the same way as the $J$-integral described above, but by replacing displacements and strains with displacement rates and strain rates:

Reply:
Should be done